app/seeds/review_seeder.rb

```diff
@@ -0,0 +1,20 @@
+class ReviewSeeder
+  REVIEWS = [
+    {
+    rider_id: 1,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.